subprojects/model-core/src/test/groovy/org/gradle/internal/reflect/validation/ValidationMessageCheckerTest.groovy

```diff
@@ -384,29 +384,6 @@ Please refer to https://docs.gradle.org/current/userguide/validation_problems.ht
 """
     }
 
-    @ValidationTestFor(
-        ValidationProblemId.UNRESOLVABLE_INPUT
-    )
-    def "tests output of unresolvableInput"() {
```

Comment:
Would it make sense to check that we fail here, or [MissingTaskDependenciesIntegrationTest.groovy](https://github.com/gradle/gradle/pull/21628/files#diff-70364cfbe4bff381eea5b7a32c02f0c2a8314df214b174f97ba8528912c292b9) already tests what is needed in this change?

Reply:
This is a unit test for them method I removed from `ValidationMessageChecker`, so I don't think it makes sense to keep it.